main.js

```diff
@@ -0,0 +1,100 @@
+// console.log("let's do this!");
+
+const Towers = function (name, pegs, discs) {
```

Comment:
cool implementation, You can review it with your mentor 💯 🥇 
If you still want it rereviewed and reevaluated, submit it again with only this file. You get 3 attempts per eval